cli/internal/cmd/upgradecheck.go

```diff
@@ -379,6 +418,10 @@ func (v *versionUpgrade) buildString() (string, error) {
 		upgradeMsg.WriteString(fmt.Sprintf("  Kubernetes: %s --> %s\n", v.currentKubernetes, strings.Join(v.newKubernetes, " ")))
 	}
 
+	if len(v.newCLI) > 0 {
+		upgradeMsg.WriteString(fmt.Sprintf("  CLI: %s --> %s\n", v.currentCLI, strings.Join(v.newCLI, " ")))
+	}
+
```

Comment:
If we print the new CLI versions like this we need to change the "No updates" [message](https://github.com/edgelesssys/constellation/pull/1394/files#diff-06df0ccd1ff71379f21375eff055cdd48759f2d6b56366e06e18fb858940a835R456). The message should not suggest to check the release page anymore and instead just state that the user is up-to-date.